clients/admin-ui/cypress/e2e/config-wizard.cy.ts

```diff
@@ -1,6 +1,7 @@
 import { stubSystemCrud, stubTaxonomyEntities } from "cypress/support/stubs";
 
-describe("Config Wizard", () => {
+// skipping while configWizardFlag exists
+describe.skip("Config Wizard", () => {
```

Comment:
I guess we should add to our "feature flag replacement" ticket (if it exists) that we need to be  able to toggle the flags in test. I think cypress mocks could accomplish that here, but not sure if it would work with the library. As is, this will mean our CI won't test the wizard during PRs while we're actively working on that feature...

Reply:
I will create the ticket now